contrib/boost-magic

```diff
@@ -13,4 +13,5 @@
 >>>30 short 17  (FairSoft jun19 or newer)
 >>>30 short 18  (FairSoft apr21 or newer)
 >>>30 short 19  (FairSoft apr22 or newer)
->>>30 short >19 (no FairSoft of today, 2024-06-12)
+>>>30 short 20  (FairSoft apr21 or newer)
```

Comment:
Based on your table in `lib/logging/BoostHelper.hpp`, I think this should be 
```suggestion
>>>30 short 20  (no FairSoft of today, 2025-07-21)
```
Reason for this: [latest Fairsoft release `Jan24p5` is still at `Boost 1.83`](https://github.com/FairRootGroup/FairSoft/blob/jan24p5/legacy/README.md#included-packages)

In case support for the newer archives is needed, this could be in fact a good reason to start preparing the next release, to be discussed with @fuhlig1